docs/features.rst

```diff
@@ -225,3 +225,18 @@ Features related to the type system
      - Not supported
      - However, an ``Input`` can receive data published by other *Connext DDS* applications that use FlatData.
 
+Loading additional Connext DDS Libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Connector* supports features that require the loading of additional *Connext DDS*
+libraries, such as
+`Monitoring <https://community.rti.com/static/documentation/connext-dds/6.0.0/doc/manuals/connext_dds/html_files/RTI_ConnextDDS_CoreLibraries_UsersManual/index.htm#UsersManual/PartMonitoringLib.htm>`__
```

Comment:
Use `current` instead of `6.0.0`. Make sure this is the right URL. Remember the Docs team changed them.

Reply:
In 6.1.0, this link should be:
https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_professional/users_manual/index.htm#users_manual/PartMonitoringLib.htm

I already changed existing links via CON-211 (the two PRs associated with this Jira have not yet been approved by Alex or merged), but if these are new links you are adding now, then you should update them here.

Reply:
I have updated this link but not the one for security plugins. How can I generate / check that I have generated the correct link?

Reply:
never mind found it below https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_secure/getting_started_guide/index.html